### PR TITLE
Sl/pallet tutorial fixes

### PR DESCRIPTION
**Pull request by lisa-parity**

### CODE REVIEW

File: content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md [hunk 1]
```diff
@@ -123,6 +123,7 @@ Therefore, the first step is to remove some files and content from the files in
 
       #[pallet::pallet]
       #[pallet::generate_store(pub(super) trait Store)]
+      #[pallet::without_storage_info]
```
Comment (shawntabrizi) [inline content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md:126]: I would prefer we put `#[pallet::unbounded]` above the storage which is bounded, or even better, make the tutorial use only bounded storage.

**Comment by shawntabrizi** [comment]
Why does this PR seem to undo everything changed here: https://github.com/substrate-developer-hub/substrate-docs/pull/1352

File: content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md (hunk 1 above)
Comment (shawntabrizi) [reply content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md:126]: I see you added `unbounded` below, which means this is not needed.

File: content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md [hunk 2]
```diff
@@ -262,54 +303,63 @@ To implement this logic in the proof-of-existence pallet:
...
+   	) -> DispatchResult {
+   		// Check that the extrinsic was signed and get the signer.
+   		// This function will return an error if the extrinsic is not signed.
+   		let sender = ensure_signed(origin)?;
+
+   		// Verify that the specified proof has been claimed.
+   		ensure!(Proofs::<T>::contains_key(&proof), Error::<T>::NoSuchProof);
+
+   		// Get owner of the claim.
+			let (owner, _) = Proofs::<T>::get(&proof).expect("This should not fail; qed");
```
Comment (shawntabrizi) [inline content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md:349]: WTF!

NO!

Never panic in the runtime! You can simply return an error here

This is also an awful proof cc @sacha-l

**Review by shawntabrizi — CHANGES_REQUESTED** [review]
big no on this one.

i have no idea what you are solving, but this PR is NOT it.

**Comment by sacha-l** [comment]
Not sure why this is being brought in - I made this branch in response to [this commit](https://github.com/substrate-developer-hub/substrate-docs/pull/1573/commits/dd4f9d6b82e357aa5525e1d03f8cef9f4f0cae83) late July and to fix other parts we'd been talking about, but then Shawn went ahead and did a whole rework of the tutorial which got merged [here](https://github.com/substrate-developer-hub/substrate-docs/pull/1352).  I should've deleted this branch, but this never should've become a PR in the first place :)

**Comment by lisa-parity** [comment]
Thanks, guys. I couldn't figure out why this was an active branch/PR.

**Pull request closed without merging**